### PR TITLE
Upgrade versions

### PR DESCRIPTION
**Pull request by mshitrit**
Upgraded SNR Tools version.

### CODE REVIEW

**Review by clobrano — APPROVED** [review]

**Review by razo7 — COMMENTED** [review]
Left a comment on using Golang v1.21 for now, except that it looks lgtm.

File: Makefile [hunk 1]
```diff
@@ -2,22 +2,22 @@
 SHELL := /bin/bash
 
 # versions at  https://github.com/kubernetes-sigs/controller-tools/releases
-CONTROLLER_GEN_VERSION = v0.12.0
+CONTROLLER_GEN_VERSION = v0.13.0
 
 # GO_VERSION refers to the version of Golang to be downloaded when running dockerized version
-GO_VERSION = 1.20
+GO_VERSION = 1.21
```
[the remark this answers is not in the document]
Comment (mshitrit) [reply Makefile:8]: Good, catch I didn't mean to upgrade to 1.21. 
It's a leftover from previous commit that I've missed.

**Pull request merged**